frontend/src/modules/layout/components/menu/workspace/menu-workspace-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <!-- Workspace list -->
+    <section v-for="(tenant, ti) of tenants" :key="tenant.id">
+      <div v-if="ti > 0" class="my-1 border-b border-gray-100" />
+      <div class="p-2">
+        <cr-menu-workspace-card class="h-14 px-3 hover:bg-gray-50" :tenant="tenant" @click="doSwitchTenant(tenant)">
+          <i v-if="currentTenant.id === tenant.id" class="ri-check-line text-lg text-black" />
+        </cr-menu-workspace-card>
+        <div v-if="currentTenant.id === tenant.id" class="pt-1 -mx-1">
+          <cr-menu-links
+            :collapsed="false"
+            :links="tenantMenu"
+            :disable-active-class="true"
+            link-class="!p-3 !h-10 !mb-0 !mt-1 !text-xs"
+          />
+          <div class="px-1">
+            <div class="p-3 h-10 text-sm text-black mt-1 rounded hover:bg-gray-50 cursor-pointer" @click.stop="emit('edit', tenant)">
+              Workspace settings
+            </div>
```

Comment:
Below tenantMenu and Wroskapce Settings there should be a border. This border should separate the menu items and the tenants list/Add workspace button

Reply:
There is a line, it was just too light